web/explorer/src/components/UploadOptions.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <Card>
+        <template #content>
+            <div class="flex flex-wrap align-items-center justify-content-center gap-3">
+                <div class="flex-grow-1 flex align-items-center justify-content-center">
+                    <FileUpload
+                        mode="basic"
+                        name="model[]"
+                        accept=".json,.gz"
+                        :max-file-size="10000000"
+                        :auto="true"
+                        :custom-upload="true"
+                        choose-label="Upload from local"
+                        @uploader="$emit('load-from-local', $event)"
+                    />
+                </div>
+
+                <Divider layout="vertical" class="hidden-mobile">
+                    <b>OR</b>
+                </Divider>
+                <Divider layout="horizontal" class="visible-mobile" align="center">
+                    <b>OR</b>
+                </Divider>
+
+                <div class="flex-grow-1 flex align-items-center justify-content-center gap-2">
+                    <FloatLabel>
+                        <InputText id="url" type="text" v-model="loadURL" />
+                        <label for="url">Load from URL</label>
```

Comment:
do we want to expose this like this? seems a rather odd use case for regular users